enterprise/utils.py

```diff
@@ -594,3 +599,10 @@ def format_price(price, currency='$'):
     if int(price) == price:
         return '{}{}'.format(currency, int(price))
     return '{}{:0.2f}'.format(currency, price)
+
+
+def get_configuration_value(val_name, default=None, **kwargs):
```

Comment:
I was going to question the `**kwargs`, but it looks like you DTRT and copied the method signature from edx-platform.

Reply:
That's right. :)